source/strings.tex

```diff
@@ -4174,6 +4180,36 @@
 and initializing \tcode{size_} with \tcode{len}.
 \end{itemdescr}
 
+\indexlibraryctor{basic_string_view}%
+\begin{itemdecl}
+template<class It, class End>
+  constexpr basic_string_view(It begin, End end);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\constraints
+\begin{itemize}
+\item \tcode{It} satisfies \tcode{contiguous_iterator},
+\item \tcode{End} satisfies \tcode{sized_sentinel_for<It>},
+\item \tcode{is_same_v<iter_value_t<It>, charT>} is \tcode{true}, and
```

Comment:
The similar change to `span` ends the `\item`s with periods.

Reply:
That was my change vs. the incoming paper; it seems a lot more readable with periods and avoids an awkward note-within-a-sentence for the very last bullet.